cloudformation/raw-invoice-textract-data-store.yaml

```diff
@@ -1,3 +1,8 @@
+AmazonTextractRawInvoiceDataTopic:
```

Comment:
Do you think we will need some kind of policy around this SNS topic? Like to say that it can invoke the RawInvoiceTextractDataQueue? 
@phinterseer what do you think?